test_runner/src/main/kotlin/ftl/filter/TestFilters.kt

```diff
@@ -73,7 +74,11 @@ object TestFilters {
         }
     }
 
-    fun fromTestTargets(targets: List<String>): TestFilter {
+    fun fromTestTargets(testTargets: List<String>) = fromTestTargets(testTargets, emptyList())
```

Comment:
Is this method necessary? Why not add a default parameter for the method below?

Reply:
old habits from java really do die hard.